src/mtype.d

```diff
@@ -4747,7 +4747,8 @@ extern (C++) final class TypeSArray : TypeArray
             if (d1 != d2)
             {
             Loverflow:
-                error(loc, "Static array size too big", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
+                auto static_arr_size = tbn.size(loc) * d2;
```

Comment:
@JohanEngelen Well, I think mulu() is exact same thing with multiplication except it checks for overflow. So the code should be fine (I could be wrong of course).